### PR TITLE
[Untracked] Add sepa/sepa instant support to mock payments

### PR DESCRIPTION
**Pull request by tgillis-circle**
Banking Circle only supports sepa and sepa_instant rails, to test mock payments on smokebox/sandbox via the sample app we need options for these new rails

### CODE REVIEW

File: pages/debug/payments/mocks/wire.vue [hunk 1]
```diff
@@ -74,7 +74,7 @@ export default class CreateMockIncomingWireClass extends Vue {
 
   currencyTypes = ['USD', 'EUR', 'SGD', 'MXN']
 
-  rails = ['wire', 'rtgs', 'spei']
+  rails = ['wire', 'rtgs', 'spei', 'sepa', 'sepa_instant']
```
Comment (grantboicheff) [inline pages/debug/payments/mocks/wire.vue:77]: are we ready for this to show in sample app for all environments?  should we limit it to test envs?

Comment (tgillis-circle) [reply pages/debug/payments/mocks/wire.vue:77]: Ah good point, I didn't realize we used this for non test envs? Do end users have access to this?

Comment (grantboicheff) [reply pages/debug/payments/mocks/wire.vue:77]: yeah we have a sample app for all envs.  You might be able to hide it behind an env flag though.

Comment (shurmanau-circle) [reply pages/debug/payments/mocks/wire.vue:77]: Where this list of rails is used?
When I go to the sample app, I see a different list of rails
image

Comment (shurmanau-circle) [reply pages/debug/payments/mocks/wire.vue:77]: Ah, we use this list of rails for https://sample-smokebox.circle.com/debug/payments/mocks/wire only. So I believe this is applicable to mocks and test env only?

Comment (svohra-circle) [reply pages/debug/payments/mocks/wire.vue:77]: @shurmanau-circle That's correct the mocks/wire is only applicable for smokebox and sandbox

Comment (svohra-circle) [reply pages/debug/payments/mocks/wire.vue:77]: Also SEPA is live for BankFrick so we can add it in all env, but sepa_instant it makes sense to just enable on smokebox / staging for testing before we roll it out to sandbox.

Comment (tgillis-circle) [reply pages/debug/payments/mocks/wire.vue]: I updated this to only show sepa_instant on smokebox and staging

**Review by svohra-circle — APPROVED** [review]

**Pull request merged**